index.js

```diff
@@ -129,6 +129,9 @@ module.exports = function(content) {
 	}
 
 	if(config.interpolate && config.interpolate !== 'require') {
+		content = content.replace(/(ng-)?pattern=(["']?\/((?:.(?!["']?\s+(?:\S+)=|[>"']))+.)\/["']?)/g, function (pattern, $1, $2) {
```

Comment:
Would it instead be possible to ignore `ng-*` in general ?

Reply:
Support ng-* pattern at new commit